### ISSUE TITLE
unable to detect Cargo.toml crate's version change

### ISSUE THREAD

**Issue by GopherJ** [rust-analyzer/rust-analyzer#7646]
I was using reqwest@0.10.8 in one project, then I start to need `.tcp_keepalive` function which is in reqwest@0.10.10, so I changed version number in Cargo.toml and save it, but rust-analyzer still reports errors `missing .tcp_keepalive`

**Comment by memoryruins** [comment]
It took a moment to to update, but it eventually does for me.

https://user-images.githubusercontent.com/6868531/107705806-52300780-6c8d-11eb-8296-780948ea1466.mp4

Did you edit the `Cargo.toml` by hand in the same editor or via a tool like `cargo-edit`? (the latter isn't detected yet)
Which editor and rust-analyzer version are you using?

**Comment by flodiebold** [comment]
Might be duplicate of #5333.

**Comment by lnicola** [comment]
@GopherJ are you still seeing this after #9426?

**Comment by GopherJ** [comment]
I think this can be closed, I didn't experience this anymore, sorry that I was busy
